src/lwc/utilPageLevelMessage/utilPageLevelMessage.js

```diff
@@ -35,13 +35,13 @@ export default class utilPageLevelMessage extends LightningElement {
     get assistiveText() {
         switch (this.variant) {
             case SUCCESS:
-                return SUCCESS;
+                return GeLabelService.CUSTOM_LABELS.commonAssistiveSuccess;
```

Comment:
Above you are assigning CUSTOM_LABELS to a variable local to the current script and then using this.CUSTOM_LABELS.commonAssistiveSuccess - any reason not to bring this into consistency?

Reply:
(I might actually have a moderate preference for doing it this way actually - using GeLabelServie.CUSTOM_LABELS - but not a strong one)